kinds/dashboard/dashboard_kind.cue

```diff
@@ -165,31 +188,71 @@ lineage: schemas: [{
 			...
 		} @cuetsy(kind="interface")
 
-		#LoadingState: "NotStarted" | "Loading" | "Streaming" | "Done" | "Error" @cuetsy(kind="enum") @grafanamaturity(NeedsExpertReview)
-
-		// FROM: packages/grafana-data/src/types/templateVars.ts
-		// TODO docs
-		// TODO what about what's in public/app/features/types.ts?
-		// TODO there appear to be a lot of different kinds of [template] vars here? if so need a disjunction
+		// Generic variable model to be used for all variable types
```

Comment:
'variable model' describes what this is in terms of the code but doesn't describe what it produces.

Perhaps this is named wrong, Variable*Model* could just be Variable, right? Then the description could read:

> A variable is a placeholder for a value. You can use variables in metric queries and in panel titles."

(stolen from https://grafana.com/docs/grafana/latest/dashboards/variables/)

Reply:
This totally makes sense. But the `VariableModel` type is widely used. I'm worried about this breaking change because it will be backported to 10.0.x, so we're putting a breaking change for a patch. @sdboyer, is it ok to introduce breaking changes in `@grafana/schema` for the next patch?